### DISCUSSION TITLE
[FEATURE-REQUEST] Posibilidad de desactivar state mal creadas dentro de la maquina de estado

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#162] [3. Ideas & Feature Requests] [answered]
Hola

Esta es una duda que lleva  hace tiempo y que me han respondido, pero lo levanto aquí como información.

Actualmente cuando se crea una maquina con 10 estados para el cliente. despues el cliente quiere 2 más y no habrá problemas, pero cuando el cliente indique que no son 12 estados sino 6, 6 te quedan "molestando", dentro del flujo y mi pregunta es: Se podrá desactivar esos estados en un futuro?. Si es por el asunto de la task, es fácil arreglar, se regulariza o se informa al cliente (en temas de implementación, y cuando el cliente se encuentra en producción es lo mismo.

Muchas gracias y atentos

**Comment by rafaelwalter** [comment, marked as answer]
@NicolasPLG actualmente no se pueden desactivar estados. Esta en el backlog del producto esa mejora, pero aun sin fecha de entrega.

**Reply by NicolasPLG** [reply]
gracias @rafaelwalter  :(

**Reply by rafaelwalter** [reply]
@NicolasPLG si necesitas priorizar esta funcionalidad hable con su representate de cotalker para coordinar la priorizacion!

**Reply by NicolasPLG** [reply]
Eso lo tengo entendido pero no es urgente, tengo otras prioridades (hoy), que se está gestionando  🗡️
